examples/rust/opensk/src/lib.rs

```diff
@@ -64,25 +67,55 @@ struct WasefireEnv {
     // c: PhantomData<C>,
 }
 
-impl AttestationStore for WasefireEnv {
-    fn get(&mut self, id: &Id) -> Result<Option<Attestation>, Error> {
-        todo!()
+impl Persist for WasefireEnv {
+    fn find(&self, key: usize) -> CtapResult<Option<Vec<u8>>> {
+        let Ok(res) = store::find(key) else {
+            // TODO: What is the correct error code to use here? There doesn't
+            // seem to be an error for value not found or something like that.
+            return Err(CTAP1_ERR_INVALID_COMMAND);
+        };
+        match res {
+            Some(g) => Ok(Some(g.into_vec())),
+            None => Ok(None),
+        }
     }
 
-    fn set(&mut self, id: &Id, attestation: Option<&Attestation>) -> Result<(), Error> {
-        todo!()
+    fn insert(&mut self, key: usize, value: &[u8]) -> CtapResult<()> {
+        // TODO: Impl `From<wasefire::Error>` for `Ctap2StatusCode`
+        if let Err(e) = store::insert(key, value) {
+            // TODO: What is the correct error code to use here? There doesn't
+            // seem to be an error for value not found or something like that.
+            // We probably need to convert `wasefire_error::Code` to `Ctap2StatusCode`.
+            Err(CTAP1_ERR_INVALID_COMMAND)
+        } else {
+            Ok(())
+        }
```

Comment:
Same here, this can just be a match.
```suggestion
        match store::insert(key, value) {
            Err(_) => Err(Ctap2StatusCode::Xxx),
            Ok(()) => Ok(()),
        }
```

Reply:
Done.